guide.md

```diff
@@ -38,53 +47,80 @@ For a complete set of options related to discovery see the [shibboleth documenta
 
 ### SimpleSAMLPhp
 
-In authsources.php (relative to the SSP config directory) find your SAML authentication source (often named 'default-sp') and set the discoURL parameter to https://use.thiss.io/ds/:
+In authsources.php (relative to the SSP config directory) find your SAML authentication source (often named 'default-sp') and set the discoURL parameter to https://service.seamlessaccess.org/ds/:
 
 ```
 'default-sp' => array(
     'saml:SP',
     'entityID' => NULL,
-    'discoURL' => 'https://use.thiss.io/ds/',
+    'discoURL' => 'https://service.seamlessaccess.org/ds/',
     ....
 ),
 ```
 
 For more details visit the [SSP documentation](https://simplesamlphp.org/docs/stable/).
 
-## Using the Login button
+---
 
-The main feature of the thiss-js software is the active login button which uses browser local store to remember and present the last Identity Provider selection made by the user. For most use-cases this completely elliminates the search/find/select UX for most interactions with your SP since most users rely on the same IdP for all their logins to a site. In order to take advantage of this feature you must do two things:
+# Implementing a "Standard" Integration
 
-  1. Display the login button component somewhere on your SP login page
-  2. Integrate the javascript API of the button component with your SAML SP software stack.
+![Illustration of the Standard integration](/assets/img/SA-full.png)
 
-The first step is easy to describe in general but the second step depends on exactly how your SAML implementation works. We provide examples below for the Shibboleth SP but if you run something other than Shibboleth you need to consult your documentation to figure out how the integration works.
+For the "Standard" flavor, you will implement:
+
+* Discovery Service Integration - _See the section above for instructions._
+* Display of Seamless Access Login Button - _Use the Seamless Access service to display the login button component on your SP login page._
+* Integration of Login Button with your SAML SP - _Integrate the Seamless Access service to use it as your Discovery Service._
+
+With the "Standard" Integration, you will be using the active login button which uses browser local store to remember and present the last Identity Provider selection made by the user. For most use-cases this completely eliminates the search/find/select UX for most interactions with your SP since most users rely on the same IdP for all their logins to a site. 
+
+## Display of Seamless Access Login Button
 
-## Displaying the Login Button
+### Including the component library
 
-Include the following HTML somewhere on your SPs login page:
+You will need to reference the Seamless Access component library before you can use it. For the Standard implementation, you will always reference through a CDN:
```

Comment:
Is this true - does it always need to be through the CDN, or can one download it using NPM, and use the installed package?

If it is possible to use a downloaded version, what might be some of the reasons to do so? Do you think we should include this in the documentation?

Finally, if we are offering a downloaded version, should we include other ways to download as well?

* Direct download
* Yarn
* Bower

Also, will we eventually need something like a Migrate plugin for version management / legacy feature management?

Reply:
Also, for the NPM version, perhaps we should remove the use documentation from the package page, and instead provide the link to the official documentation so we only have to update in one place. On the package page we can just include how to install it and how to include it in one's code. Regarding the include information, should we include more ways in the documentation than JS, ES6 & browser? For example, see the npm package page for jQuery: https://www.npmjs.com/package/jquery